### DISCUSSION TITLE
Wifi Interface loss connection

### DISCUSSION THREAD

**Discussion by Strakomania** [wlcrs/huawei_solar#610] [Setting up the integration]
I tried to connect my Huawei Inverter like MajoV from this repo https://github.com/wlcrs/huawei_solar/issues/586

I succsessfully connect 3 USB Wifi Sticks to my Homeassistant an i was able to connect them with the wifi from the inverter. The Huawei Solar Integration works pretty well. 

After a few Hours i got the Problem that one of the Wifi Sticks Disconnect from the Inverter so i have to Reboot the System. After the Reboot the WIFI Sticks will connect automatically. 

I Stuck every WIFI Stick to an Extention Cable and they placed with a distance from about 1,5m to each other. 

I hope Somebody can help me. 

![1](https://github.com/wlcrs/huawei_solar/assets/118754009/86248b6f-903f-4c4a-9e6c-9a441087b029)
![Unbenannt](https://github.com/wlcrs/huawei_solar/assets/118754009/17cb87a3-40f9-4631-85bf-fbc2c561c767)

**Comment by 730522js** [comment]
I have the same issue. I'm having the Huawei SUN2000 inverter with [SDongle FE] - I was connecting via WiFi via wlan0 interface of my HA. Connection was stable for almost 1 year untill this morning at about 6:50 CET when the connection went down and on reconnecting I get the "Failed to Change Network Settings" error in HA. However I'm able to connect to the Huawei AP from my laptop and I can also connect to other APs from the wlan0 interface of HA. I've restarted the inverter and HA several times without any effect.

I've read through documentation that if the Inverter fails to connect to the Huawei cloud then the dongle connection will also fail and there is a maintenance notification from Huawei on their servers but that's due on 1st of March...don't know if they are actually doing something else but the Fusion Solar native Huawei APP on my smartphone works properly so I would exclude any front-end-facing works on their side at this time.

Any ideas what can be the problem? Or other users experiencing this?

**Reply by Juanig** [reply]
Same here, could you fix it?

**Comment by iranzzoni** [comment]
New instalation of extensión. From the bigining , It keeps disconecting from inverter hotspot. Needs restarting everytime.
